app/controllers/api/v1/tracks_controller.rb

```diff
@@ -5,7 +5,7 @@ class TracksController < Api::BaseController
       respond_to :json
 
       def index
-        @conference ? (tracks = @conference.tracks) : (tracks = Track.all)
+        @conference ? (tracks = @conference.program.tracks) : (tracks = Track.all)
```

Comment:
OK.It was already in that form.So I assumed it was okay.I'll fix everything ASAP